clusterloader2/testing/density/suite.yaml

```diff
@@ -0,0 +1,8 @@
+- identifier: vanilla
```

Comment:
out of curiosity, where is this suite.yaml file being consumed? and how do we ensure that this only gets executed on kubemark?

Reply:
It's not used here, but we'll need to update kubernetes/test-infra's scalability tests to use the suite instead of directly running the config.yaml